doc/userguide/devguide/internals/engines/stream/applayertriggerrawreassembly.rst

```diff
@@ -0,0 +1,67 @@
+Triggering Raw Stream Reassembly from Application Layer
+-------------------------------------------------------
+
+It is possible to trigger the reassembly of the raw stream from the application
+layer protocol parsers, by calling the function
+``AppLayerParserTriggerRawStreamReassembly``.
```

Comment:
Is there also not a side effect that we run detection on this new stream data at the same time that the packet that triggered `AppLayerParserTriggerRawStreamReassembly` ?

Reply:
I don't know.. How can I verify this?

Reply:
I think that maybe IPS mode is where this becomes more evident - if I understand this correctly.

Reply:
Could you please tell if there's an example somewhere?

Reply:
This was the main one that came to my mind: https://github.com/OISF/suricata-verify/pull/1990/files#diff-f190572e1c257ec35a5a6decd12162f5852aa45c9b9c1d339593c23e7b3ca047

(but we also see something similar with DNS, IIRC)